javaharness/java/arcs/android/AndroidParticleLoader.java

```diff
@@ -0,0 +1,17 @@
+package arcs.android;
+
+import javax.inject.Inject;
+
+import arcs.api.Particle;
+import arcs.api.ParticleLoader;
+
+final class AndroidParticleLoader implements ParticleLoader {
+
+  @Inject
+  AndroidParticleLoader() {}
+
+  @Override
+  public Particle loadParticle(String particleName) {
+    return null;
```

Comment:
you've removed ParticleFactory class in favor of the Loader (which btw could have been a standalone PR as well, sorry for repeating myself, I won't any more, promise),but this doesn't do anything.
ParticleFactory wasn't used as part of the prototype, but the prototype is an incomplete use of the functionality that we want to provide. The Factory is useful for the factory particle instantiation which was used in earlier demos and would be useful again in the future. Is there a reason to remove it?

Reply:
See my previous comments about ArcsLocal. Having particles injected statically via factories implicitly mean we will have statically-linked particles and Arcs, which breaks the molecularity and makes it hard to dynamically load particles.

Reply:
But it will be the case that they'll be some 'linked into APK' particles tho? When did we decide all particles are dynamically loadable?

I suppose if we follow Shane's dream, then the linked-in code would use a Storage API only.

Reply:
I think the storage API is still TBD? We can rewrite this part if it's needed in the future. Right now it confuses me as the demo use cases only need remote particles.